devlib/instrument/gem5power.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, target, power_sites):
             self.add_channel(field, 'power')
         self.target.gem5stats.book_roi(self.roi_label)
         self.sample_period_ns = 10000000
+        self.sample_rate_hz = 1 / (self.sample_period_ns * 10**-9)
```

Comment:
As a side note, I'm not sure if this really applies to gem5. Indeed, because of gem5 behaviour, it's not guaranteed that the `Gem5PowerInstrument` provides samples at that rate. In practice, it provides samples every 10ms  **+** additional asynchronous samples for various reasons. That's actually why the instrument always provides a `time` channel. Without it, there is no way we can figure out how much time has elapsed between two samples. So, I'm just wondering if providing `sample_rate_hz` is a good idea here. It might be misleading I suppose.